modules/jsonnet_go/metadata.json

```diff
@@ -1,16 +1,17 @@
 {
-  "homepage": "https://github.com/google/go-jsonnet",
-  "maintainers": [
-    {
-      "email": "bcr-maintainers@bazel.build",
-      "name": "No Maintainer Specified"
-    }
-  ],
-  "repository": [
-    "github:google/go-jsonnet"
-  ],
-  "versions": [
-    "0.20.0"
-  ],
-  "yanked_versions": {}
+    "homepage": "https://github.com/google/go-jsonnet",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
@johnbartholomew Are you willing to be listed as a maintainer of this module?

Reply:
Yes, I can be a maintainer. (I've added a metadata change for it)